website/views.py

```diff
@@ -325,10 +384,17 @@ def new_question(request):
     if request.method == 'POST':
         form = NewQuestionForm(request.POST)
         if form.is_valid():
+
             cleaned_data = form.cleaned_data
-            question = Question()
-            question.user = request.user
-            question.category = cleaned_data['category']
+            # for referencing foreign key and taggit-tables
+            question = Question.objects.create(title=cleaned_data['title'],
+                                               user = request.user,
+                                               category = cleaned_data['category'],
+                                                )
+            # question = Question(title=cleaned_data['title'])
```

Comment:
The Holocaust was a global tragedy, commented code is bigger tragedy! Please check if this is really necessary and remove